qa/rpc-tests/txn_doublespend.py

```diff
@@ -29,118 +29,119 @@ def run_test(self):
         # All nodes should start with 1,250 BTC:
         starting_balance = 1250
         for i in range(4):
-            assert_equal(self.nodes[i].getbalance(), starting_balance)
+            print(self.nodes[i].getbalance())
```

Comment:
I think we should not commit this file since it already commented out from the test list. If we commit this changes, in the future when we enable this file we will found out it is success and we might not take a look on this file to uncomment the code.